File: exercises/ruby/flow_control/fizz_buzz/examples/002/fizz_buzz.rb
i = 1
while i <= 100
  if (i % 15 == 0)
    puts "FizzBuzz"
  elsif i % 5 == 0
    puts "Buzz"
  elsif i % 3 == 0
    puts "Fizz"
  else
    puts i
  end

  i += 1
end

File: exercises/javascript/flow_control/fizz_buzz/examples/002/fizz_buzz.js
function fizzbuzz(){
  var numbers = [];
  for (var i = 1; i <= 100; i++) {
    str = ""
    if (i % 3 == 0) { str += "Fizz" }
    if (i % 5 == 0) { str += "Buzz" }
    numbers.push(str == "" ? i : str);
  }
  console.log(numbers);

